src/modules/client/mixins/configMixin.js

```diff
@@ -39,6 +40,12 @@ export const configMixin = {
         this.tmpInput = '';
       }
     },
+    nbrError (path) {
+      if (!get(this.$v, path).required) return REQUIRED_LABEL;
```

Comment:
Ici il faut plutot que tu mettes `get(this.$v, path) === false` parce que sinon comme required n'est pas defini pour les champs non required, ca renvoie ce label.